apps/connect/src/components/document-editor-container.tsx

```diff
@@ -1,113 +1,113 @@
-import { openUrl } from '#utils';
+import { openUrl } from "@powerhousedao/connect";
 import {
-    exportFile,
-    setSelectedNode,
-    useDocumentModelModuleById,
-    useDriveIsRemote,
-    useDriveRemoteUrl,
-    useParentFolder,
-    useSelectedDocument,
-    useSelectedDrive,
-} from '@powerhousedao/reactor-browser';
-import { buildDocumentSubgraphUrl } from '@powerhousedao/reactor-browser/utils/switchboard';
-import { type PHDocument } from 'document-model';
-import { useMemo } from 'react';
-import { useTranslation } from 'react-i18next';
-import { useModal } from '../components/modal/index.js';
-import { validateDocument } from '../utils/validate-document.js';
-import { DocumentEditor } from './editors.js';
+  buildDocumentSubgraphUrl,
+  exportFile,
+  setSelectedNode,
+  useDocumentModelModuleById,
+  useDriveIsRemote,
+  useDriveRemoteUrl,
+  useParentFolder,
+  useSelectedDocument,
+  useSelectedDrive,
+} from "@powerhousedao/reactor-browser";
+import type { PHDocument } from "document-model";
+import { useMemo } from "react";
+import { useTranslation } from "react-i18next";
+import { useModal } from "../components/modal/index.js";
+import { validateDocument } from "../utils/validate-document.js";
```

Comment:
there are many places like here where these imports should be changed to use the package name import style instead of these relative imports, both for convenience and to prevent further circular import problems.